src/applications/ezr/containers/App.jsx

```diff
@@ -26,22 +36,17 @@ const App = props => {
     loading: isLoadingProfile,
   } = user;
   const isAppLoading = isLoadingFeatures || isLoadingProfile;
+  const { isUserLOA3 } = useSelector(selectAuthStatus);
   const { canSubmitFinancialInfo } = useSelector(selectEnrollmentStatus);
 
-  /**
-   * Redirect users without the prod feature toggle enabled to the VA.gov home page
-   *
-   * NOTE: this is temporary functionality while the new application is being
-   * rolled out for user research and production testing
-   */
   useEffect(
     () => {
-      if (!isLoadingFeatures && !isProdEnabled) {
-        window.location.replace('https://www.va.gov');
+      if (isUserLOA3) {
```

Comment:
Where is this logic tested? Since we changed it should some test somewhere have changed or do we currently not have coverage here?